README.md

```diff
@@ -17,15 +17,15 @@ $ dapp update
 
 If the weekly executive needs to onboard a new collateral:
 
-1. Update the `onboardNewCollaterals()` function in `Goerli-DssSpellCollateral.sol`
+1. Update the `onboardNewCollaterals()` function in `DssSpellCollateral.sol`
 2. Update the values in `src/test/config.sol`
 3. Add `onboardNewCollaterals();` in the `actions()` function in `DssSpellAction`
 
 ### Removing Collaterals from the System
 
 If the weekly executive needs to offboard collaterals:
 
-1. Update the `offboardCollaterals()` function in `Goerli-DssSpellCollateral.sol`
+1. Update the `offboardCollaterals()` function in `DssSpellCollateral.sol`
 2. Update the values in `src/test/config.sol`
 3. Add `offboardCollaterals();` in the `actions()` function in `DssSpellAction`
```

Comment:
these are outdated since we removed the separate file

Reply:
Removed it as the section doesn't actually make sense now.